### PR TITLE
update - added sql data

### PR DESCRIPTION
**Pull request by tinatsina**
## Summary
- added the SQL data, schema and queries
- pushed to GitHub

## Pictures
image
image
image
image
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,13 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/*Queries that provide answers to the questions from all projects.*/
+
+select * from animals where name like '%mon';
+select name from animals where '[2016-01-01,2019-01-01]'::daterange @> date_of_birth;
+select name from animals where neutered is true and escape_attempts < 3;
+select * from animals where name like '%Agumon' or name like '%Pikachu';
```
Comment (cynthiainga) [inline queries.sql:8]: - [ ] Here you should select the date_of_birth form animal and not select all. Please kindly fix that 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @tinatsina, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Descriptive PR ✔️ 

## Required Changes ♻️

- [ ] Please kindly update your `README` file by using this [template](https://github.com/microverseinc/readme-template)
- [ ] Please check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (tinatsina) [reply queries.sql:8]: ## Resolved
- used your advice and changed to the date.
- thanks for noticing

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,13 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/*Queries that provide answers to the questions from all projects.*/
+
+select * from animals where name like '%mon';
+select name from animals where '[2016-01-01,2019-01-01]'::daterange @> date_of_birth;
```
[the remark this answers is not in the document]
Comment (tinatsina) [reply queries.sql:6]: ## Resolved
- changed the date to Dec 31 2019
- thanks for noticing

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @tinatsina                      :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good making the required changes with regards to the db :+1: 

 ## Required Changes 
- [ ] Kindly update your readme file with the template provided by the previous reviewer. That is the official template to be used by the students.




 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @tinatsina, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**